lib/iris/tests/test_basic_maths.py

```diff
@@ -662,7 +666,15 @@ def test_operator(self):
     def test_operator_in_place(self):
         for test_op in self.iops:
             test_op(self.cube1, self.cube2)
-            test_op(self.data1, self.data2)
+            if test_op == operator.itruediv:
+                # Python3 itruediv requires floats to return floats, numpy1.10:
+                # "TypeError: ufunc 'true_divide' output (typecode 'd') could
+                # not be coerced to provided output parameter (typecode 'l')
+                # according to the casting rule ''same_kind''"
+                test_op(ma.MaskedArray(self.data1, dtype='float'),
+                        ma.MaskedArray(self.data2, dtype='float'))
```

Comment:
@corinnebosley Looks great. My only suggestion would be to reduce this change to ...

```python
test_op(data1.astype('float'), data2.astype('float'))
```
Otherwise I'm :+1: on this PR :smile: